linera-rpc/src/mass_client.rs

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use crate::RpcMessage;
+
```

Comment:
Nit: No need for the whitespace here.

Reply:
Yes indeed, we're generally trying to maximize the normalization opportunities for Rustfmt

Reply:
The blocks here are ordered by scope (ancestor < crate < workspace < external < std), so there shouldn't be any normalization opportunities between them (different crates will always end up in different blocks).  Unfortunately rustfmt doesn't understand anything about the scope of crates, so while it's smart enough to order `super` and `crate` dependencies at the top, without the newlines it will mix `std` and `linera` crates in with third-party dependencies :-\

Reply:
rustfmt has added preliminary support for this, and since we're already using nightly rustfmt I've created PR https://github.com/linera-io/linera-protocol/pull/1821 to codify import grouping behaviour.